src/components/shared/NumberInput/NumberInput.container.ts

```diff
@@ -0,0 +1,56 @@
+import styled from 'styled-components';
+import { color, typography } from 'styled/utils';
+
+interface ValueMap {
+  [key: string]: string;
+}
+
+interface Props {
+  font?: string;
+  textColor?: string;
+  end?: string;
+  start?: string;
+}
+
+const typographyMap: ValueMap = {
```

Comment:
did we decide to move these out to a utils file or somewhere where we can reuse it?

Reply:
talked in person. will make a separate ticket to condense these (this particular one is being used in 4 places)